user/reference/linux.md

```diff
@@ -0,0 +1,20 @@
+---
+title: The Ubuntu Linux Build Environments
+layout: en
+
+redirect_from:
```

Comment:
do we need the redirect from here if there's nothing to redirect from? Shall we remove it @Pezi777 ?